### PR TITLE
bug/2846_empty_maps_in_metrics

### PR DESCRIPTION
**Pull request by kzangeli**
Fixed issue #2846

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -2,6 +2,7 @@
 - Add: new option to set the timeout for REST request: -reqTimeout (#2762)
 - Fix: modified the upper limit for CLI '-maxConnections', from 1020 to UNLIMITED
 - Fix: proper 500 Internal Server Error response with descriptive text when "sort fail" at DB occur (#2752)
+- Fix: avoid to present empty metrics for services (#2846)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Given that the Metrics API has been developed as part as this sprint, this CNR line is not needed. I mean, the line:

> Add: metrics REST API (#2750)

already covers it.

**Comment by kzangeli** [comment]
Ah ... ok :-)
Long sprint ... :-)

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: Fixed in f3ac44a

**Comment by fgalan** [comment]
LGTM

**Pull request merged**